frontend/src/app/(protected)/inbox/page.tsx

```diff
@@ -0,0 +1,232 @@
+"use client";
```

Comment:
New notification page. Right now hard coded messages. Once backend is done, I'll hook it up.